style.css

```diff
@@ -0,0 +1,92 @@
+*
+{
+    padding:0px;
+    margin:0px;
+}
+#navbar
+{
+    position:fixed;
+    top:0px;
+    left:0px;
+    width:330px;
+    height:100vw;
+    background-color: lightpink;
+
+    border-right: 3px solid black;
+}
+.nav-link
+{
+    display:flex;
+    line-height: 45px;
+    font-size:large;
+    text-decoration: none;
+    border-top:3px solid black;
+}
+#navbar a
+{
+    padding-left:30px;
+    color:black;
+    font-size:larger;
+
+}
+#navbar header
+{
+    padding:15px 40px;
+}
+a:hover
+{
+    background-color: rebeccapurple;
+
+    font-weight:bolder;
+    transition: background-color 0.1ms ease-in-out;
+
+}
+#main-doc
+{
+    width:750px;
+    margin-left:400px;
+}
+main p
+{
+    width:100%;
+    line-height: 25px;
+    margin-bottom:15px;
+}
+    .main-section ul li
+    {
+        margin-top:20px;
+margin-left:50px;
+    }
+
+    code
+    {
+        padding:20px;
+        width:70%;
+        margin:30px 0px;
+        background-color: #eee;
+        display:block;
+       }
+section header
+{
+    margin:15px 0px;
+    text-shadow:2px 2px orange;
+}
+@media (max-width:600px)
+{
+
+    #navbar
+    {
+        width:200px;
+    }
+    .nav-link
+    {
+
+        font-size:10px;
+    }
+    section .main-section p,   section .main-section li
+    {
+        width:300px;
+        margin-left:-200px;
+   
+    }
+}
```

Comment:
Kindly correct your code as you assumed to do or delete it since you are not asked about responsiveness for this project